connector-common/src/main/java/com/redhat/cloud/notifications/connector/payload/PayloadOutgoingKafkaHeaderGeneratorProcessor.java

```diff
@@ -0,0 +1,24 @@
+package com.redhat.cloud.notifications.connector.payload;
+
+import com.redhat.cloud.notifications.connector.Constants;
+import com.redhat.cloud.notifications.connector.ExchangeProperty;
+import jakarta.enterprise.context.ApplicationScoped;
+import org.apache.camel.Exchange;
+import org.apache.camel.Processor;
+
+@ApplicationScoped
+public class PayloadOutgoingKafkaHeaderGeneratorProcessor implements Processor {
```

Comment:
I would say to keep it separately? Just because that way once we reactor the stuff we can simply remove the files, the processors from the routes and we're done! And also we don't mix the Cloud Event building with adding headers and what not... What do you think?

Reply:
ok then